frontend/www/js/omegaup/components/Countdown.test.ts

```diff
@@ -1,20 +1,52 @@
 import { shallowMount } from '@vue/test-utils';
 
 import omegaup_Countdown from './Countdown.vue';
+let dateNowSpy: any;
 
 describe('Countdown.vue', () => {
+  beforeAll(() => {
+    const now = Date.now();
+    // Lock Time
+    dateNowSpy = jest.spyOn(Date, 'now').mockImplementation(() => now);
+  });
```

Comment:
`now` debería ser una variable accesible a todas las pruebas:

```suggestion
  let now = Date.now();
  let dateNowSpy: ... | null = null; // aquí no se vale usar `any`.
  beforeAll(() => {
    dateNowSpy = jest.spyOn(Date, 'now').mockImplementation(() => now);
  });
```

Reply:
Listo